frontend/src/components/ContributorsCard/ContributorCard.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Card from "@material-ui/core/Card";
+import CardActions from "@material-ui/core/CardActions";
+import CardContent from "@material-ui/core/CardContent";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+import Avatar from "@material-ui/core/Avatar";
+
+const useStyles = makeStyles({
+  root: {
+    // minWidth: 275,
+    maxWidth: 400,
+    padding: 10,
+    margin: 10,
+    border:'10px'
+  },
+  bullet: {
+    display: "inline-block",
+    margin: "0 2px",
+    transform: "scale(0.8)",
+  },
+  title: {
+    fontSize: 14,
+  },
+  pos: {
+    marginBottom: 12,
+  },
+});
+
+const ContributorsCard = () => {
+  const [listOfContributors,setListOfContributors] = useState([]);
+  useEffect(()=>{
+      fetch("https://api.github.com/repos/shravan20/github-readme-quotes/contributors?")
```

Comment:
Can you move the URL to a urlConfig.js file and export it from there under a new folder called /constants?? This is to ensure that we maintain code quality. Please make the changes by evening, we will have it accepted by evening.

Reply:
okay no problem in that, can i make the folder inside the Contributors Card or do you want it to be in the components?

Reply:
Create a folder /frontend/src/CONSTANTS. 
Create a file urlConfig.js. Just move URL to that!
 
Rest of the things looks neat!

Reply:
I have exported the URL from a new file, kindly check the changes in the latest commit